addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/internal/server/ServerConnection.java

```diff
@@ -31,11 +30,11 @@
 /**
  * Holds information about connection to Minecraft server.
  *
- * @author Mattias Markehed
+ * @author Mattias Markehed - Initial contribution
  */
 public class ServerConnection {
 
-    private static final Logger logger = LoggerFactory.getLogger(ServerConnection.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServerConnection.class);
```

Comment:
Does it need to be static?

Reply:
If possible not is what I mean :-)

Reply:
I've managed to move the logger to the object where it is used so it is no longer static.